pkg/plugin/interfaces.go

```diff
@@ -56,7 +56,7 @@ type Subcommand interface {
 	Run() error
 	// InjectConfig passes a config to a plugin. The plugin may modify the config.
 	// Initializing, loading, and saving the config is managed by the cli package.
-	InjectConfig(*config.Config)
+	InjectConfig(config.Config)
```

Comment:
The original intent of passing a pointer to a config was to allow the plugin to modify the config before the CLI wrapper could make modifications/write the config back to disk, and more generally propagate plugin-specific changes to the caller. I'd prefer to keep that functionality.

Reply:
`config.Config` is an interface now

Reply:
Whoops I did not notice that. Thanks for clarifying.